NMEA2000.h

```diff
@@ -345,6 +349,13 @@ inline void SetN2kProductInformation(tN2kMsg &N2kMsg, unsigned int N2kVersion, u
                   SertificationLevel,LoadEquivalency);
 }
 
+
+void SetN2kPGN59904(tN2kMsg &N2kMsg, uint8_t Destination, unsigned long RequestedPGN);
+
+inline void SetN2kPGNISORequest(tN2kMsg &N2kMsg, uint8_t Destination, unsigned long RequestedPGN) {
```

Comment:
Here is a 'back-door' resource I am using for J1939:  the RV-C spec  ([http://www.rv-c.com/?q=node/75](http://www.rv-c.com/?q=node/75))    Like NMEA2000, it is built on top of J1939,and as such it documents several of the underlying foundation messages, like Requests and addresses, etc.

ref: 3.2.4.3 #1 concerning how to respond of a Request.